languages.toml

```diff
@@ -214,6 +214,16 @@ roots = []
 comment-token = ";"
 indent = { tab-width = 4, unit = "    " }
 
+[[language]]
+name = "csharp"
+scope = "source.csharp"
+injection-regex = "csharp"
+file-types = ["cs"]
+roots = []
+comment-token = "//"
+indent = { tab-width = 4, unit = "    " }
+language-server = { command = "/home/yoeight/tools/omnisharp/run", args = ["-z", "-s", "$PWD", "--encoding", "utf-8", "--loglevel", "info"] }
```

Comment:
You don't have to specify the directory; it'll run the LSP if it's in PATH.

Reply:
I know that but as I said in the first comment, Omnisharp has different binary names depending on the platform. For example on Linux, the binary is named `run`. This is too generic to rely on its presence in $PATH

https://github.com/helix-editor/helix/pull/604#issuecomment-900347635

Reply:
There's no good way to script that yet sadly. I think this will have to be up to the user config:

https://github.com/neovim/nvim-lspconfig/blob/62902d21fae90f2b008913074246debec850f2ae/lua/lspconfig/omnisharp.lua#L25-L36